integration/simulation/validate_chain.go

```diff
@@ -285,33 +278,46 @@ func checkBlockchainOfObscuroNode(
 }
 
 // FindNotIncludedL2Txs returns the number of transfers and withdrawals that were injected but are not present in the L2 blockchain.
-func FindNotIncludedL2Txs(l2Client rpcclientlib.Client, txInjector *TransactionInjector) (int, int) {
+func FindNotIncludedL2Txs(nodeIdx int, rpcHandles *network.RPCHandles, txInjector *TransactionInjector) (int, int) {
```

Comment:
@BedrockSquirrel I think it makes sense to pass in the client, as we did before, rather than selecting the client (twice) in this method?